### ISSUE TITLE
2-coding - repetitive?

### ISSUE THREAD

**Issue by amromeo** [pcmathias/AACC-Intro-to-R-2023#5]
concepts of functions, objects and arguments are really drilled in, too much?

**Comment by pcmathias** [comment]
I don't think this can be drilled in too much. There were still a couple who weren't completely comfortable with assigning to objects until they did multiple exercises.
